tatsu-lab/alpaca_farm
# [ENH] migrate annotator to alpaca eval

Planning / have to do:
- [x] migrate annotators to alpaca eval
- [x] inherit from alpaca eval to split instruction/input (need to make changes upstream to simplify inheritance)
- [x] update configs to work with alpaca eval
- [x] update notebooks
- [x] test code for eval and annotations
- [x] test variance (should be smaller than before)
- [x] rerun the minimal leaderboard 

Thins that I would ideally do but not planning to:
- use the open AI generation from alpaca_eval instead of code duplication
- remove `alpaca_leaderboard` and use alpaca_eval functions

closes https://github.com/tatsu-lab/alpaca_eval/issues/11
